src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -175,18 +195,17 @@ public GenerateScriptCommand(OctoLogger log, ISourceGithubApiFactory sourceGithu
             }
         }
 
-        private async Task<string> InvokeGithub(string githubSourceOrg, string githubTargetOrg, string ghesApiUrl, string azureStorageConnectionString, bool noSslVerify, bool sequential)
+        private async Task<string> InvokeGithub(string githubSourceOrg, string githubTargetOrg, string ghesApiUrl, string azureStorageConnectionString, bool noSslVerify, bool sequential, string githubSourcePat)
         {
-            var targetApiUrl = "https://api.github.com";
-            var repos = await GetGithubRepos(_sourceGithubApiFactory.Create(targetApiUrl), githubSourceOrg);
+            var repos = await GetGithubRepos(_sourceGithubApiFactory.Create(sourcePersonalAccessToken: githubSourcePat), githubSourceOrg);
```

Comment:
Why don't we need to pass in a url here anymore? Because the PAT has is enough information? I've seen this in other places as well, so this question is more of a general one.